docs/advanced-entity-analytics/entity-risk-scoring.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[entity-risk-scoring]]
+= Entity Risk Scoring
+
+beta::[]
+
+Entity Risk Scoring is an advanced {elastic-sec} analytics feature that helps security analysts detect changes in an entity's risk posture, hunt for new threats, and prioritize incident response.
+
+Entity Risk Scoring allows you to monitor the change in the risk posture of hosts and users from your environment. The risk scoring engine generates these advanced scoring analytics by factoring threats from its end-to-end XDR use cases, such as SIEM, cloud, and endpoint.
+
+The next-generation risk scoring engine provides greater scalability and performance. It leverages the Elastic SIEM detection engine to generate host and user risk scores from the last 30 days.
+
+It also generates risk scores on a recurring interval, and allows for easy onboarding and management. The engine is built to factor in risks from all {elastic-sec} use cases, and allows you to customize and control how and when risk is calculated.
```

Comment:
```suggestion
It also generates risk scores on a recurring interval, and allows for easy onboarding and management. The engine is built to factor in risks from all {elastic-sec} use cases, and allows you to customize and control how and when risk is calculated.
```
"data from... use-cases" seems off to me. Maybe replace with "Data from various sources" or "Data from across SIEM"?

Reply:
I'm not sure which sentence this suggestion refers to?